src/views/AssetsView.module.css

```
@@ -0,0 +1,583 @@
+  z-index: 7;
+}
+
+/* ── Screen reader only live region ─────────────────────────────── */
+.srOnly {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)